ui/src/tool-panel/location-track/split-store.ts

```diff
@@ -14,15 +14,21 @@ import { getPlanarDistanceUnwrapped } from 'map/layers/utils/layer-utils';
 
 const DUPLICATE_MAX_DISTANCE = 1.0;
 
-export type InitialSplit = {
+type SplitBase = {
     name: string;
     descriptionBase: string;
     suffixMode: LocationTrackDescriptionSuffixMode;
     duplicateOf?: LocationTrackId;
     location: Point;
+    new: boolean;
```

Comment:
Tässä tiketin varsinainen pihvi. Tämän perusteella focusataan uuden splitin nimi-kenttä, ideana se että ka. splitti merkataan välittömästi tämän jälkeen vanhaksi.

Ikävää että tämän joutuu säilömään täällä, mutta muutakaan paikkaa meillä ei oikein tälle ole, eikä splitin uutuutta voi oikein päätelläkään mistään piirtoaikaisesti. Uudet splitit lisätään reduxin kautta, joten splittien merkkaaminen siellä uusiksi menee hyvin näppärästi. Huonona puolena se, että jos splittien vanhaksi merkkaaminen koskaan hajoaa, niin splitit varastavat aina focuksen.

Reply:
Mulle itselleni tämä ei ole niin issue, plus me rikotaan muualla tätä paljon esim. `switch`-termin kanssa. Voin tosin muuttaa tämän johonkin muuhun muotoon jos `new`:in karttaminen saa enemmän kannatusta.

Reply:
Kumsi-kumsa... Osassa meidän boolean fieldejä on isJotain ja osassa ei. Yleisesti sanoisin että kenttänimessä ei kannata olla is mukana, koska java-konvention mukaisesti boolean fieldien gettereissä sana get vaihtuu sanaksi is. Eli isNew olisi vähän vastaava kuin kentän nimenä olisi getNew, mikä olisi tietenkin hassua.

Sanoisin että jos keksit paremman nimen kuin new (tarkoittaako tämä muuten new-as-in-latest vai new-as-in-unedited?) niin käyttäisi mielummin sitä, mutta is-liitettä en siihen lisäisi.